config/core/deployments/webhook-hpa.yaml

```diff
@@ -43,7 +43,7 @@ metadata:
   labels:
     eventing.knative.dev/release: devel
 spec:
-  minAvailable: 80%
+  minAvailable: 1
```

Comment:
Part of the problem is discussed here:
https://github.com/knative/operator/issues/376

/cc @nak3 @davetropeano

Reply:
Isn't the same argument applicable to activator-pdb?